client/lib/analytics/statsd.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import startsWith from 'lodash/startsWith';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+export function statsdUrl( path, eventType, duration, currentPath = undefined ) {
```

Comment:
Also this featureSlug business seems tightly coupled to the previous placeholder waiting observer code.